src/pdf/pdf-document.js

```diff
@@ -4,18 +4,49 @@ export const FATHER_ADDRESS = 'fatherAddress';
 export const CAREGIVER_ADDRESS = 'caregiverAddress';
 
 let createDocDefinition = (inputInfo, childName) => {
+  console.log("DATA IN PDF FILE BUUUM ^___^", inputInfo);
   const statusOne = inputInfo.parentalStatus === '0' ? 'X' : '__';
   const statusTwo = inputInfo.parentalStatus === '1' ? 'X' : '__';
   const statusThree = inputInfo.parentalStatus === '2' ? 'X' : '__';
   const statusFour = inputInfo.parentalStatus === '3' ? 'X' : '__';
   const docDefinition = {
     footer: (currentPage, pageCount) => {
+      if(currentPage < 4){
+        return;
+      }
       return {
-        text: currentPage.toString() + ' of ' + pageCount,
+        text: `${currentPage - 3} of ${pageCount - 3}`,
         alignment: 'center'
       };
     },
     content: [
+      {
+        text: 'POWER OF ATTORNEY FOR CARE OF A MINOR CHILD Insctruction TITLE SAMPLE\n\n',
+        alignment: 'center',
+        bold: true,
+        decoration: 'underline'
+      },
+      {
+        // blank page
+        text: '',
+        style: 'header',
+        pageBreak:'before'
+      },
+      {
+        text:'Instroction should be here An instruction document is a step-by-step guide on how to perform an activity. Instruction templates are purposeful templates. They are not for leisure reading. People read them for purpose and they are worthy of it. They are more about ‘how’ rather than ‘what’. There are many kinds of instruction templates: You may also see'
```

Comment:
What is the purpose of this new section? The PDF example pinned in Slack starts with the title and the first paragraph on line 58.

Reply:
Max clarified that we are adding an intro section and creating one PDF with blank pages between each child's section.